### ISSUE TITLE
Nitpick - `rebecca-purple` isn't a predefined color

### ISSUE THREAD

**Issue by jacobofbrooklyn** [brownplt/pyret-lang#1675]
Hey, a more experienced programmer here, just saw this and wanted to try it out. 

I was playing around with colors and wanted to use the color `rebeccapurple` (a valid CSS color). Well, it seems like it doesn't exist. I wouldn't be upset if it weren't for the fact that *it seems to be the ONLY COLOR missing from [the predefined colors](https://www.pyret.org/docs/latest/color.html#%28part._s~3acolor-constants%29)* that is present as a default CSS color.

What? Why? How on earth did this slip through?? Not sure if this is worth fixing or if it's maybe intended for some reason, but it sure is confusing!
Kind of making this as a joke but my mind has been thoroughly boggled by this LOL!

**Comment by jacobofbrooklyn** [comment]
Yeah I wrote [a little JavaScript code](https://jsitor.com/xxhlqBW3fg) to confirm my suspicions. What I found: `rebeccapurple` is indeed the only CSS color missing from the predefined colors list, and there are a couple of predefined colors that aren't included as CSS colors:

CSS colors not included in predefined: `rebeccapurple`
Predefined colors that aren't CSS colors: `violetred`, `mediumgoldenrod`, `mediumforestgreen`, `transparent`

**Comment by shriram** [comment]
Okay, I have NEVER seen that name before, that is impressive!

**Comment by jpolitz** [comment]
We may need to ask someone from the world of Racket (but back when it was called PLT Scheme). Our list is essentially this list for historical reasons, by way of WeScheme and Danny Yoo/@schanzer:

https://docs.racket-lang.org/draw/color-database___.html

**Comment by sorawee** [comment]
It sounds like a (relatively) recent addition, according to https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/

**Comment by blerner** [comment]
If I recall correctly, I mostly pulled the names from the X11 color list, not the CSS color list.  The overlap between those lists is pretty thorough, though.  There's no reason not to add it.

**Comment by jacobofbrooklyn** [comment]
So do you think this should be added? I would be willing to contribute a fix for this since it seems like a relatively easy change (although I have no idea what the codebase is like).

**Comment by jpolitz** [comment]
If someone in the community thought it was important and made a PR to add
it, I'd support it :-)

(Not least because tiny PRs like this are often good documentation
artifacts to have)
